main/webapp/modules/core/scripts/project.js

```diff
@@ -175,35 +176,58 @@ Refine.setTitle = function(status) {
 };
 
 Refine.reinitializeProjectData = function(f, fError) {
-  $.getJSON(
-    "command/core/get-project-metadata?" + $.param({ project: theProject.id }), null,
-    function(data) {
+  Promise.all([
+    Promise.resolve($.getJSON("command/core/get-project-metadata", { project: theProject.id }))
+    .then(data => {
       if (data.status == "error") {
-        alert(data.message);
-        if (fError) {
-          fError();
```

Comment:
```
       if (fError) {
          fError();
```

I'm curious, why stop using this mechanism?

Reply:
I didn't. I just moved it to the `catch()`